### PR TITLE
[11.0][ADD] partner_children_navigation: Allow browsing from a contact form view to its contacts

### PR DESCRIPTION
**Pull request by dajuayen**
To navigate from a customer or supplier to its contacts,
the name of the contact in the kanban card of each contact
in the contacts and addresses tab in the form view of the source contact is a link
that when clicked on it, shows us the information of the selected contact.

To return to the form with the data of the source contact, you just have to retrace the path of the bread crumbs.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The module name shouldn't contain plurals. I would go for `partner_children_navigation`.

**Comment by dajuayen** [comment]
> The module name shouldn't contain plurals. I would go for `partner_children_navigation`.

@pedrobaeza 
'children' is a word with a plural meaning.
Is it okay for you to rename the module as 'partner_child_navigation'?

**Comment by pedrobaeza** [comment]
@Dajuayen the problem is not with words that implies plural (like `children` or `people`). The specific problem is with `s` for making a plural. It happens that you misspell continuously module names due to these `s`. For example, previously the module `partner_multi_relation` was named `partner_relations`. Do you see the change I want to express? Check also out contributing section about that: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id6

**Comment by dajuayen** [comment]
@pedrobaeza thanks for the explanation. I have already renamed the module and the pull request.

**Comment by pedrobaeza** [comment]
@Dajuayen look how I rename the PR title for adjusting it to guidelines.

**Comment by dajuayen** [comment]
@pedrobaeza I think everything is ok.

**Comment by pedrobaeza** [comment]
Why do you rename again the title? I have put it the proper way.

**Comment by dajuayen** [comment]
Sorry @pedrobaeza , I did not understand your message well. I thought you were asking me to change the title of the PR to finish approving it.

On the other hand, everything is OK, isn't it?

**Comment by pedrobaeza** [comment]
@Dajuayen I have only made a review on the PR guidelines, not the content itself.

File: partner_children_navigation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# SDI
+# © 2018 David Juaneda <djuaneda@sdi.es>
```
Comment (NL66278) [inline partner_children_navigation/__manifest__.py:2]: Use the word Copyright here, not the symbol.

File: partner_children_navigation/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# SDI
```
Comment (NL66278) [inline partner_children_navigation/__manifest__.py:1]: Delete this line

File: partner_children_navigation/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# © 2018 David Juaneda <djuaneda@sdi.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Navigate to a partner's contacts",
+    'version': '11.0.1.0.0',
+    'development_status': 'Beta',
+    'category': 'Partners',
+    'website': 'https://github.com/OCA/social',
```
Comment (NL66278) [inline partner_children_navigation/__manifest__.py:9]: website should be .../partner-contact

File: partner_children_navigation/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# SDI
+# © 2018 David Juaneda <djuaneda@sdi.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline partner_children_navigation/__manifest__.py:3]: Make it https:

File: partner_children_navigation/models/res_partner.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# SDI
```
Comment (NL66278) [inline partner_children_navigation/models/res_partner.py:1]: Delete first line, use Copyright word, https: url....

**Comment by pedrobaeza** [comment]
Isn't this the same as https://odoo-community.org/shop/product/partner-contact-access-link-5422 ?

File: partner_children_navigation/tests/__init__.py [hunk 6]
```diff
@@ -0,0 +1 @@
+from . import test_res_partner
```
Comment (NL66278) [inline partner_children_navigation/tests/__init__.py:1]: Make sure there is the newline.

File: partner_children_navigation/tests/test_res_partner.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
+# SDi
```
Comment (NL66278) [inline partner_children_navigation/tests/test_res_partner.py:1]: See comments on manifest and model file...

**Review by NL66278 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please take up the comments given.

And black your python code, this will minimize changes when upgrading to future releases. Blacked code will be required for Odoo >= 13.0.

**Comment by pedrobaeza** [comment]
@NL66278 please see https://github.com/OCA/partner-contact/pull/649#issuecomment-698876179

**Comment by NL66278** [comment]
@pedrobaeza It looks the same. So shall I close this? If functionality needed on 11.0, the 12.0 module should be backported I think...

**Comment by pedrobaeza** [comment]
Yes, I think so.

**Comment by NL66278** [comment]
This module provides the same functionality as partner_contact_access_link in 12.0 and 13.0. So, if the functionality is also needed on 11.0, backport that module.

**Pull request closed without merging**